docs/tutorials/creating-a-spec.rst

```diff
@@ -84,7 +84,7 @@ our grid we get:
 Masking with Regions
 --------------------
 
-We can apply a `Mask` to only include positions within a given `Region` using
+We can apply a `Mask` to only include points within a given `Region` using
```

Comment:
And here too. Maybe we should rename middle -> midpoints?

Reply:
As discussed in the meeting, all instances of 'positions'/'middle' have been updated to 'midpoints'. 
Half tempted to change the lower and upper terms to lower_bounds having read through the code. But I'll let you review it first in case its okay as it is.

Reply:
```suggestion
We can apply a `Mask` to only include frames where the midpoints are within a given `Region` using
```